waku/node/v2/waku_types.nim

```diff
@@ -70,7 +71,7 @@ type
     switch*: Switch
     rng*: ref BrHmacDrbgContext
     peers*: seq[HistoryPeer]
-    messages*: seq[WakuMessage]
```

Comment:
Why is messages removed here?

1) What if I don't want to use a DB
2) Wouldn't there be a "cache" anyway for easy access?

Reply:
1. If you don't you don't pass a path and sqlite is only used in memory.
2. Hmm not sure. Going off of status-go it didn't really do that either.

Reply:
What if I don't have SQLite and dont want it or can't use it for some reason? To me it should still work with in memory naive data structure.  No reason to remove this capability.

Reply:
don't have sqlite? it's included in the binary, assuming you start using nim-sqlite3-abi instead of the std library sqlite module (which can be assumed to be broken)